### PR TITLE
INF-2227: Add documentation for COManage CO Group provisioning.

### PR DESCRIPTION
**Pull request by williamnswanson**

### CODE REVIEW

**Review by mwestphall — APPROVED** [review]
I was able to follow this document to add a new COManage Group, looks good.

**Review by brianhlin — CHANGES_REQUESTED** [review]
I've got thoughts on formatting but will have to provide details at a later time

File: docs/operations/comanage-recipes.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+COManage Recipes
+==============================
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:9]:
```suggestion
```

All of this basically just repeats what you say in the sentence above. Also, adding a section level here adds noise to the section hierarchy and it's not entirely clear why we need it (if we have another section in this doc at the same level, what would it actually be?)

File: docs/operations/comanage-recipes.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### Provisioning a CO Group in COManage
+In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),
+ it must first be provisioned.
+
+Follow these steps to provision a CO Group into LDAP:
+
+1. #### Create CO Group in COManage and add members (or use existing CO Group)
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:17]:
```suggestion
### Create CO Group in COManage and add members (or use existing CO Group)
```

I don't think we need to make something a section header AND an enumerated step

File: docs/operations/comanage-recipes.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    Skip the first bullet point if using an existing Group.
+
+    -   Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:21]:
```suggestion
    1.  **If using an already existing group, skip to the next step.** Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.
```

Combining and bolding here helps with clarity. Also, this section has two steps that should happen in order (if doing both). We use bulleted lists more when we're describing exclusive options, or an unordered list

File: docs/operations/comanage-recipes.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        ![Create-Group-pt1](../img/comanage-create-group.png)
+        Give the group a name then click `ADD`, which will bring you to the Edit page for the new group
+        ![Create-Group-pt2](../img/comanage-group-edit-page.png)
+    -   Click on `MEMBERS`, then type in the name or identifier for a user you want to give membership to, 
+        then select the user from the drop-down and click the `ADD` button. 
+        Repeat as necessary for each group member.
+        As the creator of the Group, you will already have both Membership in, and Ownership over, the new group.
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:28]:
```suggestion
        As the creator of the Group, you will already have both Membership in and Ownership over the new group.
        Remove yourself as appropriate.
```

File: docs/operations/comanage-recipes.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
+
+### Provisioning a CO Group in COManage
+In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:12]:
```suggestion

In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on hosts),
```

Adding a linebreak between the heading and content for readability, not sure why hosts was capitalized

File: docs/operations/comanage-recipes.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+ it must first be provisioned.
```
Comment (brianhlin) [inline docs/operations/comanage-recipes.md:13]: I think it'd be worth adding an admonition here to speak to the fact that OIDC clients needing groups don't need any of these steps outside of the group creation (see https://osg-htc.org/technology/documentation/style-guide/#notes and https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)

```
```

**Review by brianhlin — COMMENTED** [review]
See in-line comments. Mostly nit-picky stuff to adhere to our style guide: https://osg-htc.org/technology/documentation/style-guide/.  Make sure to batch commits together when accepting suggestions (https://osg-htc.org/technology/software/git-software-development/#batch-commits-when-accepting-suggestions-from-a-review)

Also as a heads up, you should basically never submit PRs from your main / master branches as the default PR behavior is to allow the maintainer to add commits to your PR, i.e. your fork's main / master branch

File: docs/operations/comanage-recipes.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (williamnswanson) [reply docs/operations/comanage-recipes.md:28]:
```suggestion
        As the creator of the group you will already have both Membership in, and Ownership over, the new group.
        
        Remove yourself as appropriate.
```
Github won't let me add two suggestions for the same line, so I'm just making one suggestion with both fixes (capitalization and only having one type of comma in a sentance).

**Pull request closed without merging**